region/maridia/inner-green/Oasis.json

```diff
@@ -894,44 +1188,179 @@
       ],
+      "flashSuitChecked": true,
       "note": [
-        "Use Screw Attack to break the bomb block by entering from a non-water room with a spin jump, and make it to the top.",
-        "It helps to enter as low as possible and with as much horizontal speed as possible and with HiJump turned off."
+        "Use Screw Attack to break the bomb block by entering from a non-water room with a last-frame spin jump, and make it to the platform below the top door.",
+        "Longer runway lengths can also work, up to 17 tiles, but will make the trick somewhat more precise.",
+        "Use angle to break spin, and do a momentum conserving turnaround either on the same frame or one frame later;",
+        "this first turnaround should be done early enough that the turnaround is complete by the time Samus clears the two-tile passage in the middle of the room.",
+        "Do a second momentum conserving turnaround somewhat late, and by switching from holding left to holding right one frame later (a perfect 'boomerang')."
       ],
-      "devNote": [
-        "This does not have collision oscillation",
-        "FIXME: It is possible to use a flash suit to get from 4 through the top door, but either requires a diagonal spark,",
-        "or using this strat to get onto the top platform, which is more than this strat expects."
-      ]
+      "detailNote": [
+        "The optimal amount of extra run speed is between $3.2 and $3.5, by using 13 tiles of runway or about a half tile more or less;",
+        "this gives a 7-frame window for the first turnaround, and typically a 4-frame window for the second turnaround if the first turnaround is done early enough.",
+        "Extra run speeds between $3.6 and $3.F can also work, but with shorter windows.",
+        "A perfect boomerang is always required for the second turnaround, regardless of where in the window it is done.",
+        "It is recommended to buffer the spin break, by holding angle (up or down), jump, and forward through the transition,",
+        "then switching from forward to backward, with at most 1 frame of neutral in between, while still holding jump and angle."
+      ],
+      "devNote": "This does not have collision oscillation."
     },
     {
-      "id": 20,
+      "link": [1, 4],
+      "name": "Cross Room Jump To Top Door (Blue Speed)",
+      "entranceCondition": {
+        "comeInGettingBlueSpeed": {
+          "length": 0,
+          "openEnd": 1,
+          "minExtraRunSpeed": "$3.2",
+          "maxExtraRunSpeed": "$3.F"
+        }
+      },
+      "requires": [
+        {"notable": "Cross Room Jump Through Top Door"},
+        "canInsaneJump",
+        "canCrossRoomJumpIntoWater",
+        "canMomentumConservingTurnaround",
+        "h_trickyToCarryFlashSuit"
+      ],
+      "unlocksDoors": [
+        {"types": ["super"], "requires": []},
+        {"types": ["missiles", "powerbomb"], "requires": ["never"]}
+      ],
```

Comment:
Should this be tested against; or is there meaning in this?
4 doesn't have a door and there is no nodeID on the unlockDoors requirements.